.github/workflows/github-actions-stale.yml

```diff
@@ -0,0 +1,25 @@
+name: Mark stale issues and pull requests
+on:
+  schedule:
+    - cron: '0 23 * * *' # once a day at 11pm UTC time zone
+jobs:
+  stale:
+    permissions:
+      issues: write # for commenting on an issue and editing labels
+      pull-requests: write # for commenting on a PR and editing labels
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          # timing
+          days-before-stale: 60 # 60 days of inactivity
+          days-before-close: 7 # 7 more days of inactivity
+          # labels to watch for, add, and remove
+          only-labels: 'pending info' # only mark issues/PRs as stale if they have this label
+          labels-to-remove-when-unstale: 'pending info' # remove label when unstale -- should be manually added back if information is insufficient
+          # automated messages to issue/PR authors
+          stale-issue-message: 'This issue is marked as stale because it has been open 60 days with no activity. Remove pending info & stale labels or comment if you want to address this issue otherwise this will be closed in 7 days.'

```

Comment:
Would it be better to review the message wordings with doc specialists?

Reply:
@taz-pega-work  @kingakowalska1 request to review and provide your feedback on the stale/closure messages added for github issues/pr's which doesn't have any activity for 60 days will be marked as stale and it will be closed if user doesn't respond with in 30 days after making its as stale or inactive.

please review lines from 22-25 in (https://github.com/pegasystems/pega-helm-charts/pull/756/files#diff-d58a3a4335db7fb83cb5105f8e99209d37e19bc76418557d6822e7f24e27e1a9R22)